lib/workers/repository/updates/generate.spec.ts

```diff
@@ -657,5 +657,37 @@ describe('workers/repository/updates/generate', () => {
       expect(res.pendingChecks).toBeUndefined();
       expect(res.upgrades).toHaveLength(1);
     });
+    it('displays pending versions', () => {
+      const branch = [
+        {
+          depName: 'some-dep',
+          groupName: 'some-group',
+          branchName: 'some-branch',
+          prTitle: 'No pending version',
+        },
+        {
+          depName: 'some-dep',
+          groupName: 'some-group',
+          branchName: 'some-branch',
+          prTitle: 'One pending version',
+          pendingVersions: ['1.1.0'],
+        },
+        {
+          depName: 'some-dep',
+          groupName: 'some-group',
+          branchName: 'some-branch',
+          prTitle: 'Two pending versions',
+          pendingVersions: ['1.1.0', '1.1.1'],
+        },
+      ];
+      const res = generateBranchConfig(branch);
+      expect(res.upgrades.map((u) => u.displayPending)).toMatchInlineSnapshot(`
+        Array [
+          "",
+          "\`1.1.0\`",
+          "\`1.1.1\` (+1)",
+        ]
+      `);
+    });
```

Comment:
Looks like we should use `toStrictEqual` here 😉 
```suggestion
      expect(res.upgrades.map((u) => u.displayPending)).toStrictEqual(
        [
          "",
          "\`1.1.0\`",
          "\`1.1.1\` (+1)",
        ]
      );
    });
```

maybe needs prettify